tests/ci/run_formal_verification.sh

```diff
@@ -10,14 +10,12 @@ rm -rf aws-lc-verification-build
 git clone https://github.com/awslabs/aws-lc-verification.git aws-lc-verification-build
 
 cd aws-lc-verification-build
-# We avoid pulling the saw-script submodule since the repo will take forever to clone.
 git submodule update --init
-pushd Coq/fiat-crypto; git submodule update --init --recursive; popd
```

Comment:
This line is no longer needed?

Reply:
That's correct. The fiat-crypto stuff are for Coq proofs, which are only run in AWS-LC-verification CI but not in AWS-LC. 

For a little bit of history, AWS-LC CI used to recursively fetch all submodules in AWS-LC-verification when setting up the run. The Coq/saw-script submodule is very large and is not used in AWS-LC CI. So in this commit https://github.com/aws/aws-lc/commit/fd686fac4581a2c70a277907876c7c40a345d569, it is made sure that only Coq/fiat-crypto is fetched but not Coq/saw-script. I realized that Coq/fiat-crypto is also not needed for the SAW proofs, so I removed this line.